convmv.md

```diff
@@ -0,0 +1,7 @@
+# convmv
+
+> Converts filenames from one encoding to another
+
+- convert encoding without dry run
```

Comment:
"without dry" run is confusing. The default invocation does a dry run in addition to the conversion, or instead of it? Maybe add an example above this one without the `--notest`, to better clarify the command's behavior.